### PR TITLE
knowledge: fix html character encoding

### PR DESCRIPTION
**Pull request by hachreak**
* Fixes html character encoding on admin template.

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: invenio/modules/knowledge/templates/knowledge/list.html [hunk 1]
```diff
@@ -25,13 +25,13 @@
     </li>
     {% if admin_view.can_create %}
     <li>
-        <a href="{{ get_url('.create_view', url=return_url, kbtype='t') }}" title="{{ _('Create new record "Taxonomy"') }}">{{ _('Create KB "Taxonomy"') }}</a>
+        <a href="{{ get_url('.create_view', url=return_url, kbtype='t') }}" title="{{ _('Create new record &#34;Taxonomy&#34;') }}">{{ _('Create KB "Taxonomy"') }}</a>
```
Comment (lnielsen) [inline invenio/modules/knowledge/templates/knowledge/list.html]: I'm not sure if it's smart to have HTML entities in translation strings. Perhaps better write a title without need of the quotes or alternatively, change from ``title=""`` to ``title=''``

File: invenio/modules/knowledge/templates/knowledge/list.html [hunk 2]
```diff
@@ -25,13 +25,13 @@
...
     </li>
     <li>
-        <a href="{{ get_url('.create_view', url=return_url, kbtype='d') }}" title="{{ _('Create new record "Dynamic"') }}">{{ _('Create KB "Dynamic"') }}</a>
+        <a href="{{ get_url('.create_view', url=return_url, kbtype='d') }}" title="{{ _('Create new record &#34;Dynamic&#34;') }}">{{ _('Create KB "Dynamic"') }}</a>
     </li>
     <li>
-        <a href="{{ get_url('.create_view', url=return_url, kbtype='w') }}" title="{{ _('Create new record "Written As"') }}">{{ _('Create KB "Writtes As"') }}</a>
+        <a href="{{ get_url('.create_view', url=return_url, kbtype='w') }}" title="{{ _('Create new record &#34;Written As&#34;') }}">{{ _('Create KB "Writtes As"') }}</a>
```
Comment (lnielsen) [inline invenio/modules/knowledge/templates/knowledge/list.html]: ``Writtes`` -> ``Written``

File: invenio/modules/knowledge/templates/knowledge/list.html (hunk 1 above)
Comment (jirikuncar) [inline invenio/modules/knowledge/templates/knowledge/list.html]: you can use `|e` (http://jinja.pocoo.org/docs/dev/templates/#escape)

**Comment by hachreak** [comment]
Summary on `escape` usage:

If you put: 
```
 title="{{ _('Create new record "Taxonomy"') | e }}">
```
It doesn't work!
Hml generated:
```html
title="Create new record "Taxonomy"">
```
See a demo on python shell:

```python
In [1]: from markupsafe import Markup 
In [2]: from jinja2.filters import escape
In [3]: escape('Create new record "Taxonomy"')
Out[3]: Markup(u'Create new record &#34;Taxonomy&#34;')
In [4]: escape(Markup('Create new record "Taxonomy"'))
Out[4]: Markup(u'Create new record "Taxonomy"')
```

**Pull request closed without merging**